test/unit/spec/signaling/v2/peerconnection.js

```diff
@@ -32,6 +32,34 @@ describe('PeerConnectionV2', () => {
     });
   });
 
+  describe('.isApplicationSectionNegotiated', () => {
+    context('when the underlying RTCPeerConnection has a local RTCSessionDescription', () => {
+      [true, false].forEach(hasApplicationSection => {
+        context(`when the RTCSessionDescription ${hasApplicationSection ? 'has' : 'does not have'} an application m= section`, () => {
+          it(`should be set to ${hasApplicationSection}`, async () => {
+            const test = makeTest({
+              offers: [
+                makeOffer({ application: hasApplicationSection })
+              ]
+            });
+            // TODO(mmalavalli): Remove this line once VIDEO-1981 is fixed.
```

Comment:
https://issues.corp.twilio.com/browse/VIDEO-1981 is closed now. can this TODO be removed?